JSDemos/Demos/TreeList/LoadDataOnDemand/Angular/app/app.component.ts

```diff
@@ -29,7 +29,9 @@ export class AppComponent {
         }
         return lastValueFrom(
           http.get('https://js.devexpress.com/Demos/Mvc/api/treeListData', { params }),
-        );
+        ).then((result: any) => ({
+          data: result,
+        }));
```

Comment:
Seems that it's unnecessary in frameworks where we use `http.get` or `fetch`

@DevExpress/devextreme-grids, what do you think, should we keep it the same for all demos?

Reply:
Angular provides its [client HTTP API](https://angular.io/guide/http). We try to show a native approach for each framework.